### ISSUE TITLE
Package breaks in Meteor 1.7

### ISSUE THREAD

**Issue by kdekooter** [nicolaslopezj/roles#30]
```
ReferenceError: _ is not defined
    at new Roles.Role (packages/nicolaslopezj:roles/roles.js:89:3)
    at roles.js (packages/nicolaslopezj:roles/roles.js:321:19)
```

**Comment by banjerluke** [comment]
Fixed in version 2.6.6

**Comment by kdekooter** [comment]
Great, thanks. Now bumping into https://github.com/orionjs/orioncms/issues/452...
